service/project.go

```diff
@@ -236,6 +239,15 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if projectVars.GithubHookID != 0 && projectRef.TracksPushEvents &&
+		responseRef.ForceRepotrackerRun {
+		j := units.NewRepotrackerJob(fmt.Sprintf("%d", job.GetNumber()), projectRef.Identifier)
```

Comment:
can this be "ui-triggered-job-%d"

Reply:
will do

Reply:
done; also changed the other repo tracker job to have a prefix.